### ISSUE TITLE
Suggestion: CI/CD Pipelines

### ISSUE THREAD

**Issue by abcdan** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#505]
I think it's really important for us to automatically test and deploy across multiple nodes.

**Comment by lukasvdberk** [comment]
Also cover multiple regions because you need to test with different IP's addresses and hosting companies to ensure the quality of the product

**Comment by abcdan** [comment]
Exactly. Deploy across multiple AWS/Azure/Google Cloud/etc. regions to create a stable network.

**Comment by lukasvdberk** [comment]
Do not forget to include buzz words to hype up the product and upper management
